release_notes.md

```diff
@@ -1,3 +1,45 @@
+**********************************
+# Release Notes for OpenMDAO 3.9.2
+
+May 14, 2021
+
+OpenMDAO 3.9.2 is a patch release that fixes for bugs found in 3.9.1.
```

Comment:
grammar "fixes for bugs"  ->  reword like "fixes bugs", or "provides fixes for bugs"

Reply:
got it